client/common/common.c

```diff
@@ -598,6 +607,15 @@ menu_with_options(struct client *client)
 enum bm_run_result
 run_menu(const struct client *client, struct bm_menu *menu, void (*item_cb)(const struct client *client, struct bm_item *item))
 {
+    if (!client->multi_instance_mode) {
+        int menu_lock = open("/tmp/bemenu.lock", O_CREAT | O_RDWR | O_CLOEXEC, 0666); // Create a read-write lock file(if not already exists), closes on exec. Creates in tmp due to permissions.
+        errno = 0;
+
+        if (flock(menu_lock, LOCK_EX | LOCK_NB) == -1) {  // If we cannot set the lock/the lock is already present.
+            errx(EXIT_FAILURE, "bemenu instance is already running");
```

Comment:
Lets replace this with 
```
fprintf(stderr, "bemenu instance is already running\n");
exit(EXIT_FAILURE);
```
forgot this project does not use err.h and that header does not exist on macos